lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -39,21 +43,15 @@ def initialize(term_config)
       #     "http://schema.org/name":["Cornell University","Ithaca (N.Y.). Cornell University"],
       #     "http://www.w3.org/2004/02/skos/core#altLabel":["Ithaca (N.Y.). Cornell University"],
       #     "http://schema.org/sameAs":["http://id.loc.gov/authorities/names/n79021621","https://viaf.org/viaf/126293486"] } }
-      def find(id, language: nil, replacements: {}, subauth: nil, format: nil, jsonld: false, performance_data: false) # rubocop:disable Metrics/ParameterLists, Metrics/MethodLength
-        # TODO: When jsonld parameter is removed, the format parameter should default to 'json'.  Not making this change now for backward compatibility of the default for jsonld parameter.
+      def find(id, request_header: {}, language: nil, replacements: {}, subauth: nil, format: 'json', performance_data: false) # rubocop:disable Metrics/ParameterLists
+        request_header = build_request_header(language: language, replacements: replacements, subauth: subauth, format: format, performance_data: performance_data) if request_header.empty?
+        subauth = request_header.fetch(:subauthority, nil)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data term sub-authority #{subauth}" unless subauth.nil? || term_subauthority?(subauth)
-        @language = language_service.preferred_language(user_language: language, authority_language: term_config.term_language)
+        @language = language_service.preferred_language(user_language: request_header.fetch(:language, nil), authority_language: term_config.term_language)
         @id = id
-        @performance_data = performance_data
-        @format = format
-        @jsonld = jsonld if @format.blank?
```

Comment:
The changes in this PR and another one coming soon will require a major release change.  So support of the deprecated jsonld parameter is being removed.  There will be more deprecation removals before the major release, but they will come in a separate PR.  This was removed now because of the parameter changes happening in this PR.